exercises/practice/protein-translation/tests/protein_translation.cairo

```diff
@@ -1,330 +1,195 @@
+    assert_eq!(proteins("UUUUUU"), array![Proteins::Phenylalanine, Proteins::Phenylalanine]);
 }
 
 #[test]
 #[ignore]
 fn sequence_of_two_different_protein_codons_translates_into_proteins() {
-    let mut info = parse(make_pairs());
-    assert_eq!(info.of_rna("UUAUUG"), Option::Some(array!["leucine", "leucine"]),);
+    assert_eq!(proteins("UUAUUG"), array![Proteins::Leucine, Proteins::Leucine]);
 }
 
 #[test]
 #[ignore]
 fn translate_rna_strand_into_correct_protein_list() {
-    let mut info = parse(make_pairs());
     assert_eq!(
-        info.of_rna("AUGUUUUGG"), Option::Some(array!["methionine", "phenylalanine", "tryptophan"]),
+        proteins("AUGUUUUGG"),
+        array![Proteins::Methionine, Proteins::Phenylalanine, Proteins::Tryptophan]
     );
 }
 
 #[test]
 #[ignore]
 fn translation_stops_if_stop_codon_at_beginning_of_sequence() {
-    let mut info = parse(make_pairs());
-    assert_eq!(info.of_rna("UAGUGG"), Option::Some(array![]),);
+    assert_eq!(proteins("UAGUGG"), array![]);
 }
 
 #[test]
 #[ignore]
 fn translation_stops_if_stop_codon_at_end_of_two_codon_sequence() {
-    let mut info = parse(make_pairs());
-    assert_eq!(info.of_rna("UGGUAG"), Option::Some(array!["tryptophan"]),);
+    assert_eq!(proteins("UGGUAG"), array![Proteins::Tryptophan]);
 }
 
 #[test]
 #[ignore]
 fn translation_stops_if_stop_codon_at_end_of_three_codon_sequence() {
-    let mut info = parse(make_pairs());
-    assert_eq!(info.of_rna("AUGUUUUAA"), Option::Some(array!["methionine", "phenylalanine"]),);
+    assert_eq!(proteins("AUGUUUUAA"), array![Proteins::Methionine, Proteins::Phenylalanine]);
 }
 
 #[test]
 #[ignore]
 fn translation_stops_if_stop_codon_in_middle_of_three_codon_sequence() {
-    let mut info = parse(make_pairs());
-    assert_eq!(info.of_rna("UGGUAGUGG"), Option::Some(array!["tryptophan"]),);
+    assert_eq!(proteins("UGGUAGUGG"), array![Proteins::Tryptophan]);
 }
 
 #[test]
 #[ignore]
 fn translation_stops_if_stop_codon_in_middle_of_six_codon_sequence() {
-    let mut info = parse(make_pairs());
     assert_eq!(
-        info.of_rna("UGGUGUUAUUAAUGGUUU"),
-        Option::Some(array!["tryptophan", "cysteine", "tyrosine"]),
+        proteins("UGGUGUUAUUAAUGGUUU"),
+        array![Proteins::Tryptophan, Proteins::Cysteine, Proteins::Tyrosine]
     );
 }
 
 #[test]
 #[ignore]
-fn unknown_amino_acids_not_part_of_a_codon_can_t_translate() {
-    let mut info = parse(make_pairs());
-    let none: Option<Array<ByteArray>> = Option::None;
-    assert_eq!(info.of_rna("XYZ"), none);
+fn sequence_of_two_non_stop_codons_does_not_translate_to_a_stop_codon() {
+    assert_eq!(proteins("AUGAUG"), array![Proteins::Methionine, Proteins::Methionine]);
 }
 
 #[test]
 #[ignore]
-fn incomplete_rna_sequence_can_t_translate() {
-    let mut info = parse(make_pairs());
-    let none: Option<Array<ByteArray>> = Option::None;
-    assert_eq!(info.of_rna("AUGU"), none);
+#[should_panic(expected: ("Invalid codon",))]
+fn non_existing_codon_cant_translate() {
```

Comment:
This test is reimplemented by the following test so we can remove it. If tests.toml is up-to-date, it should already be marked as not included.